src/core/server/legacy_compat/legacy_platform_proxifier.ts

```diff
@@ -50,12 +63,27 @@ export class LegacyPlatformProxifier extends EventEmitter {
         return [
           eventName,
           (...args: any[]) => {
+            // Since it's just a proxy and the underlying server is managed by the core
```

Comment:
Hmm, what inspired this change? I feel like the default behavior of falling over when something isn't listening for the `error` event is generally desired, but this now listens for that event and swallows it if nothing else is listening for it...

Reply:
I think I was hit by this during testing/development at some point, when core http server failed before legacy Kibana subscribed to the proxifier events. So the original error has been handled by Hapi 17 and core, but nothing was listening for the "forwarded" error event so we got unhandled exception.

Legacy Kibana only attaches `error` handler when Hapi starts to listen so it may take some time.

What do you think?

Reply:
I think I need more time to think this through, and maybe we could chat about it, but I think if something fails before the Legacy Kibana server attaches its error handler we probably shouldn't be swallowing it and ignoring it.

Reply:
Okay, I'll remove it then and we'll see how it goes.

Reply:
Removed :heavy_check_mark: We can get back to it if it becomes a problem.